command/utils/command_helper.cc

```diff
@@ -26,7 +27,18 @@ uint32_t CommandHandler(const string& regist, const string& user,
   } else if (regist == "" && user != "" && chirp == "" && follow != "" &&
              reply == "" && read == "" && monitor == false) {
     return kFOLLOW;
-  } else {
+  } else if (regist == "" && user !="" && chirp == "" && follow == "" &&
```

Comment:
I wish there were a better way to do this, but you're not responsible for re-writing this sequence of if statements...